[2021-02-18 22:37:03,464] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:33:00+00:00 [queued]>
[2021-02-18 22:37:03,469] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:33:00+00:00 [queued]>
[2021-02-18 22:37:03,470] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:37:03,470] {taskinstance.py:1018} INFO - Starting attempt 1 of 2
[2021-02-18 22:37:03,470] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:37:03,480] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): get_tweets_by_user_list> on 2021-02-18T22:33:00+00:00
[2021-02-18 22:37:03,482] {standard_task_runner.py:51} INFO - Started process 17851 to run task
[2021-02-18 22:37:03,487] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'tweets_by_user', 'get_tweets_by_user_list', '2021-02-18T22:33:00+00:00', '--job-id', '1114', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tweets_by_user.py', '--cfg-path', '/tmp/tmpukst213e']
[2021-02-18 22:37:03,490] {standard_task_runner.py:76} INFO - Job 1114: Subtask get_tweets_by_user_list
[2021-02-18 22:37:03,519] {logging_mixin.py:103} INFO - Running <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:33:00+00:00 [running]> on host ip-172-31-3-59.ec2.internal
[2021-02-18 22:37:03,541] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=tweets_by_user
AIRFLOW_CTX_TASK_ID=get_tweets_by_user_list
AIRFLOW_CTX_EXECUTION_DATE=2021-02-18T22:33:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-18T22:33:00+00:00
[2021-02-18 22:37:03,543] {logging_mixin.py:103} INFO - Autenticação no twitter
[2021-02-18 22:37:03,558] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:37:03,726] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:37:04,256] {logging_mixin.py:103} INFO - 200
[2021-02-18 22:37:04,265] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:37:04,438] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:37:04,919] {logging_mixin.py:103} INFO - 400
[2021-02-18 22:37:04,928] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:37:05,082] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:37:05,631] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:37:05,634] {logging_mixin.py:103} INFO - 0:00:02.091996
[2021-02-18 22:37:05,634] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:37:07,601] {logging_mixin.py:103} INFO - 0:00:01.966616
[2021-02-18 22:37:07,626] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:37:07,647] {logging_mixin.py:103} INFO - 0:00:00.046441
[2021-02-18 22:37:07,648] {logging_mixin.py:103} INFO - 600 devidamente inseridas no banco
[2021-02-18 22:37:07,650] {python.py:118} INFO - Done. Returned value was: None
[2021-02-18 22:37:07,653] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=tweets_by_user, task_id=get_tweets_by_user_list, execution_date=20210218T223300, start_date=20210218T223703, end_date=20210218T223707
[2021-02-18 22:37:07,675] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-18 22:37:07,713] {local_task_job.py:118} INFO - Task exited with return code 0
